src/Features/Core/Portable/ExternalAccess/Pythia/Api/PythiaSymbolSorting.cs

```diff
@@ -13,10 +13,9 @@ internal static class PythiaSymbolSorting
     {
         public static ImmutableArray<TSymbol> Sort<TSymbol>(
             ImmutableArray<TSymbol> symbols,
-            ISymbolDisplayService symbolDisplayService,
```

Comment:
❗️ This is a partner API, so the signature cannot change.

Reply:
Yup.  Undone.